CONTRIBUTING.md

```diff
@@ -1,63 +1,63 @@
 # Contributing
 
-Thank you so much for contributing to 1Password. We appreciate your time and help. Here are some guidelines to help you get started.
+Thanks for your interest in contributing to the 1Password Connect Terraform Provider project! 🙌 We appreciate your time and effort. Here are some guidelines to help you get started.
 
 ## Building
 
-To build the 1Password Connect Terraform provider run the following:
+Run the following command to build the 1Password Connect Terraform Provider:
 
 ```sh
-$ go build .
+go build .
 ```
 
 This will create the `terraform-provider-onepassword` binary.
 
 ## Testing the Provider
 
-To run the go tests and check test coverage run the following:
+To run the Go tests and check test coverage run the following:
 
 ```sh
-$ go test -v ./... -cover
+go test -v ./... -cover
 ```
 
 ## Generating Documentation
 
-Documentation is generated for the provider using the [terraform documentation plugin](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
+Documentation is generated for the provider using [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
 
-To regenerate the `/docs` markdown run:
+To regenerate the `/docs` Markdown run:
 
 ```sh
-$ go generate
+go generate
 ```
 
-## Installing Locally
+## Installing plugin locally
```

Comment:
Maybe move this section to be very first as this is the first thing that contributors will look for?

Reply:
Reordered sections in the [f1394fe](https://github.com/1Password/terraform-provider-onepassword/pull/98/commits/f1394fe9b19aa052678ab23e512c385467f093c7) commit.